docs/_posts/2000-01-06-docker-swarm.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Docker Swarm"
+bg: edge_water
+color: black
+fa-icon: img/docker_swarm_icon.png
+---
+
+### Before starting
+
+Setup a [docker swarm](https://docs.docker.com/get-started/part4/) with at least two nodes.
+
+#### Constellations
+
+* One manager and multiple workers. The hub will run on the manager and the 
+created browser containers will run on the workers.
+* A high available docker swarm with multiple managers. The manager that runs the hub will 
+not run any created browser container.
+
+_Info:_ Currently we do not support running created browser containers on the same node as
```

Comment:
I think it could be better if you could share the problem (logs, your investigation result, etc) so the community could help.
How do you think, @yosserO , @tstern ? 
Maybe an separate issue will be great :)

Reply:
Yes the idea is good. Currently we didn't really investigated the issue due to getting the docker swarm implementation run first.
I'll check the issue this week and setup issues in this repository for known problems.

Reply:
So far, I am happy with current Docker Swarm implementation. However, keeping the manager node stand for mostly free is not the good way to utilize resources, which is the reason we choose to go with Docker Swarm. That's why I asked this. I'll try to take a look in my free time if I can find anything helpful.
Thanks for your response, @tstern

Reply:
We're able to resolve the problem with deploying browser containers on the same node as the hub is running. It wasn't a implementation or zalenium error. The problem was our local development environment with a low-performance node running the hub. The node was overloaded by running hub and browser tests simultaneously. The high-performance docker swarm for our production didn't has this problems.
I added a pull request with the adaption: https://github.com/zalando/zalenium/pull/987

Currently we still see two problems which I'll add this week to this repo issues:
- using third-party container for docker-swarm overall executions
- logs and videos are not stored